skbio/stats/power.py

```diff
@@ -49,7 +49,6 @@
     subsample_power
     subsample_paired_power
     confidence_bound
-    bootstrap_power_curve
```

Comment:
What I mean here is adding this line back so that the docs still link to the deprecated function. Sorry if this wasn't clear before.